examples/README.md

```diff
@@ -11,3 +12,7 @@ Inside each example directory:
 1. `poetry install`
 2. `poetry run python manage.py migrate # initialize the project`
 3. `HONEYCOMB_API_KEY=api-key HONEYCOMB_DATASET=django-example poetry run python manage.py runserver`
+   
+For Tornado example, just do
```

Comment:
Just because I'm now allergic to the word "just", which can be interpreted badly, a suggestion:

```suggestion
For Tornado example, there's no need to run the migrate step. Do only this:
```